apps/communications/contacts/test/marionette/list_test.js

```diff
@@ -79,28 +75,55 @@ marionette('Contacts > List', function() {
       });
     }
 
-    test('after scrolling we keep the images', function() {
-      // Scroll to different sections in the list
-      var scrollbar = client.helper.waitForElement(
-        Contacts.Selectors.scrollbar);
-      //F
-      actions.press(scrollbar, 15, 100).release().perform();
-      client.helper.waitForElement(Contacts.Selectors.overlay);
-      client.helper.waitForElementToDisappear(Contacts.Selectors.overlay);
-      //T
-      actions.press(scrollbar, 15, 300).release().perform();
-      client.helper.waitForElement(Contacts.Selectors.overlay);
-      client.helper.waitForElementToDisappear(Contacts.Selectors.overlay);
-      //X
-      actions.press(scrollbar, 15, 350).release().perform();
-      client.helper.waitForElement(Contacts.Selectors.overlay);
-      client.helper.waitForElementToDisappear(Contacts.Selectors.overlay);
-      //A
-      actions.press(scrollbar, 15, 40).release().perform();
-      client.helper.waitForElement(Contacts.Selectors.overlay);
-      client.helper.waitForElementToDisappear(Contacts.Selectors.overlay);
+    function firstContactHasDefaultImage() {
+      return client.executeScript(function() {
```

Comment:
I don't think you need to inject a script. IMO, this can be done entirely with Marionette's API. Something like:
```js
function firstContactHasDefaultImage() {
    try {
        var span = client.findElement('#contacts-list-A .contact-item span');
        return span.cssProperty('background-image') === 'url("app://communications' +
                      '.gaiamobile.org/contacts/style/images/Imagery.png")';
    } catch (e) {
        return false;
    }
}
```

I'm not exactly sure if `cssProperty` works with `'background-image'`. If it doesn't, we can use [getAttribute](http://mozilla-b2g.github.io/marionette-js-client/api-docs/classes/Marionette.Element.html#method_getAttribute)

Also, I think we should check if the default letter is displayed in addition to the picture. What do you think? In the python test [we did it this way](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_default_contact_image.py#L31). `image_data_group` [is defined as](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py#L140).